### PR TITLE
Delete the last active key on every ROSA run

### PR DESCRIPTION
**Pull request by krishvoor**
If any, delete the second aws iam key

Signed-off-by: Krishna Harsha Voora <krvoora@redhat.com>

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi @mukrishn @morenod @chentex ptal

**Review by chentex — APPROVED** [review]
LGTM

File: dags/openshift_nightlies/scripts/install/rosa.sh [hunk 1]
```diff
@@ -157,11 +157,10 @@ setup(){
...
         echo "Clean-up existing OSD access keys.."
-        DELETE_KEYS=$(aws iam list-access-keys --user-name OsdCcsAdmin | awk '{print$2}')
-        for i in $DELETE_KEYS
-        do
-            aws iam delete-access-key --access-key-id $i --user-name OsdCcsAdmin
-        done
+        aws_key=$(aws iam list-access-keys --user-name OsdCcsAdmin --output text --query 'AccessKeyMetadata[1].AccessKeyId')
+        if [[ $aws_key != None ]]; then
+            aws iam delete-access-key --user-name OsdCcsAdmin --access-key-id $aws_key
+        fi
```
Comment (dry923) [inline dags/openshift_nightlies/scripts/install/rosa.sh:163]: I'd recomend we do something more like this so we delete the oldest key (the one at index 0) instead
```
aws_key=(`aws iam list-access-keys --user-name OsdCcsAdmin --output text --query 'AccessKeyMetadata[*].AccessKeyId'`)
if [[ ${#aws_key[@] -eq 2 ]]; then
  del aws_key[0]
fi
````

Comment (krishvoor) [reply dags/openshift_nightlies/scripts/install/rosa.sh:164]: Agreed, we should delete the last active key when both are active.

Comment (dry923) [reply dags/openshift_nightlies/scripts/install/rosa.sh:164]: len_aws_key should be ${len_aws_key}

**Review by chentex — APPROVED** [review]

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**